src/Symfony/Component/Decorator/composer.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "symfony/decorator",
+    "type": "library",
+    "description": "Decorate the functionality of other functions",
```

Comment:
```suggestion
    "description": "Decorate the functionality of other codes",
```

?

Reply:
`codes` is too broad, `functions` is a bit more specific and aligned to the concept.